ui/src/components/PopInput/PopInput.jsx

```diff
@@ -117,80 +79,84 @@ export default class PopInput extends React.Component {
   }
 
   render() {
-    const linkClass = 'editable-click';
-    const busyVisibility = this.isBusy() ? '' : 'visibility-hidden';
-    const inputVisibility = !this.isBusy() ? '' : 'visibility-hidden';
-    const title = (this.props.title === '') ? this.props.name : this.props.title;
-
-    let stateClass = 'value-label-enter-success';
```

Comment:
This class was adding a `background-color` CSS transition, which made the UI feel less responsive. I'm removing the class.